### PR TITLE
Using Kubeflow-tfjob SDK to deploy TFJob

### LINKED ISSUE #434: Using kubeflow-tfjob to deploy TFJob
/kind feature

**Describe the solution you'd like**
The `kubeflow-tfjob` SDK has been implemented and published: https://pypi.org/project/kubeflow-tfjob/

We need to enhance `kubeflow-fairing` to use kubeflow-tfjob to deploy TFJob.

From @jlewi comments in Email:

```
Jinchi and Animesh

Any thought on how the auto-generated python libraries should relate to fairing? In particular the deployers?
https://github.com/kubeflow/fairing/tree/master/kubeflow/fairing/deployers

Do you think we can replace these deployers with the auto-generated libraries? Or do you see them as higher level libraries ontop of the auto generated libraries?
```

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by jinchihe**
**What this PR does / why we need it**:
`Kubeflow-tfjob` SDK has been published, we should use the SDK to generate and deploy TFJob, see more in #434 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #434 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/435)

### CODE REVIEW

**Comment by jinchihe** [comment]
/assign @abhi-g 

Would you please take a look? Thanks.

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name='kubeflow-fairing',
-    version='0.7.0.1',
+    version='0.7.1',
```
Comment (abhi-g) [inline setup.py]: Would it make sense to separate out the "version up" from this PR?

**Comment by abhi-g** [comment]
Thanks @jinchihe. The change itself looks good. One minor comment above

File: setup.py (hunk 1 above)
Comment (jinchihe) [reply setup.py]: @abhi-g Thanks! Updated.

**Comment by jinchihe** [comment]
@abhi-g That's ready for another reviewing. Thanks a lot!

**Pull request merged**